control/control_tests.cmake

```diff
@@ -15,8 +15,11 @@ if(WITH_${COMPONENT}_TESTING)
   # Simulation tests
   BEGIN_TEST(src/tests)
   
-  NEW_TEST(tests PIDTest.cpp SMCTest.cpp ObserverTest.cpp)
+  NEW_TEST(tests)
   
+  IF(WITH_FORTRAN)
+    NEW_TEST(PIDTest.cpp SMCTest.cpp ObserverTest.cpp)
+  ENDIF()
   END_TEST()
```

Comment:
This is a bit misleading: it should be WITH_EVENTDRIVEN. We only have fortran code that perform an event-driven integration. If we have SUNDIALS bindings, then we can use the latter for performing ED integration.

Ideally I would switch inside the test to a TS OSI if no ED OSI are available.